src/components/header/ProfileThreadHeaderBar.js

```diff
@@ -95,39 +125,75 @@ class ProfileThreadHeaderBar extends PureComponent {
       threadName,
       processDetails,
       isHidden,
+      isModifyingSelection,
     } = this.props;
+
     if (isHidden) {
       // If this thread is hidden, render out a stub element so that the Reorderable
       // Component still works across all the threads.
       return <li className="profileThreadHeaderBarHidden" />;
     }
+
+    const processType = thread.processType;
+    const displayJank = thread.name === 'GeckoMain' && processType !== 'plugin';
+    const displayTracingMarkers =
```

Comment:
This logic looks new. We might get people confused and complaining about their markers not showing up.

Reply:
It's not new; it was here before: https://github.com/devtools-html/perf.html/blob/ad2c5a919f3a2646a63657436cfb4d0a8482fb3d/src/components/header/ProfileViewerHeader.js#L121 https://github.com/devtools-html/perf.html/blob/ad2c5a919f3a2646a63657436cfb4d0a8482fb3d/src/components/header/ProfileViewerHeader.js#L140-L143